vllm/model_executor/models/bloom.py

```diff
@@ -278,6 +278,41 @@ def forward(
         hidden_states = self.ln_f(hidden_states)
         return hidden_states
 
+    def load_weights(self, weights: Iterable[tuple[str,
+                                                   torch.Tensor]]) -> set[str]:
+        params_dict = dict(self.named_parameters(remove_duplicate=False))
+        loaded_params: set[str] = set()
+        for name, loaded_weight in weights:
+            if name == "lm_head.weight":
```

Comment:
This should in AutoWeightsLoader skip_prefixes to filter, like https://github.com/vllm-project/vllm/blob/9214e60631a79506e7669650de87806a123e0b0b/vllm/model_executor/models/olmo.py#L383-L391

Reply:
ok，updated